salt/modules/gpfs.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
+'''
+Module to provide GPFS compatibility to Salt.
+'''
+from __future__ import absolute_import
+
+# Import python libs
+import logging
+import os
+
+# Import salt libs
+import salt.utils
+
+log = logging.getLogger(__name__)
+
+
+def __virtual__():
+    '''
+    Verify gpfs is installed.
+    '''
+    os.environ["PATH"] += os.pathsep + '/usr/lpp/mmfs/bin'
```

Comment:
Is it possible to install this to a different location?

Could it be changed to be a config option?

Thanks,
Daniel

Reply:
Also, since this is the same with other services.  It is probably best to add docs that specify how to configure the PATH in an /etc/systemd/system dropin file, or in /etc/default/salt to add this path, instead of hard coding the path and explicitly adding it to the environment.

Because this will add this path to all users, even if they don't have this tool installed.

Reply:
This path is static, and cannot be changed, and has been like this for many years for this app. It cannot be customisable either.

I could instead check to see if that file exists, and not add it to PATH.

I am already using the full paths for the commands, so shouldn't make a difference there

Reply:
Yeah, i would rather just check if the full path is there, instead of mangling the PATH variable for all of salt just to use which to see if the a command is there.